AudioTranscriber.py

```diff
@@ -15,10 +15,12 @@
 MAX_PHRASES = 10
 
 class AudioTranscriber:
-    def __init__(self, mic_source, speaker_source):
+    def __init__(self, mic_source, speaker_source,language):
+        self.preffered_language = language
         self.transcript_data = {"You": [], "Speaker": []}
         self.transcript_changed_event = threading.Event()
-        self.audio_model = whisper.load_model(os.path.join(os.getcwd(), 'tiny.en.pt'))
+        self.audio_model = whisper.load_model(os.path.join(os.getcwd(), 'tiny'))
```

Comment:
tiny.en.pt file exists in the root directory but tiny does not exist so this will crash on this line

Reply:
We also still want to use the tiny.en. model for English because of its better performance.